### PR TITLE
In 152464564 refactor screening decision

### PR DESCRIPTION
**Pull request by saraOSI**
### Pivotal Story

- [Implement Redux validations and state management for **Decision** card #152464564](https://www.pivotaltracker.com/story/show/152464564)

### Purpose
Tackle the show half of the screening decision refactor. Edit state to come.

### CODE REVIEW

File: app/javascript/containers/screenings/DecisionShowContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  let props = {
+    accessRestriction: {
+      value: _.capitalize(getScreeningSelector(state).get('access_restrictions')),
+    },
+    additionalInformation: {
+      value: getScreeningSelector(state).get('additional_information'),
+    },
+    decision: getDecisionSelector(state).toJS(),
+    decisionDetail: getDecisionDetailSelector(state).toJS(),
+    restrictionRationale: {
```
Comment (jgedeon-osi) [inline app/javascript/containers/screenings/DecisionShowContainer.jsx]: Should this be restrictionsRationale

File: spec/features/screening/decision_spec.rb [hunk 2]
```diff
@@ -119,7 +119,7 @@
         'Determine Decision and Response Time by using Structured Decision Making'
       )
       expect(page).to have_content('Complete SDM')
-      expect(page).to have_content('Screening Decision')
+      expect(page).to have_content('Screening decision')
```
Comment (jgedeon-osi) [inline spec/features/screening/decision_spec.rb]: Reason we changed the case of this?

Comment (saraOSI) [reply spec/features/screening/decision_spec.rb]: I can verify with Daniel, but I am 95% certain a lot of Pierre's work was standardizing the capitalization of field headers, etc. There was some inconsistency in the labeling of these fields, so I standardized them.

File: app/javascript/containers/screenings/DecisionShowContainer.jsx (hunk 1 above)
Comment (saraOSI) [reply app/javascript/containers/screenings/DecisionShowContainer.jsx]: I think the field has a bad label, so I changed the code to reflect that you can only have 1 restriction. I want to check with product before changing the label on the form, but I think we should do that, too.

**Pull request merged**